docs/tinlake/overview/pricing_rwa.md

```diff
@@ -1 +1,78 @@
+The value of an invoice is firstly based on the buyer’s ability and willingness to pay. In some cases, this can be represented by publicly available credit ratings. In others, credit bureau data, the payment history of the buyer or publicly available news may be used.
+
+### 2. Supplier Credit Risk
+The stability of the supplier is another important factor to be considered. Lenders are underwriting the supplier as much as the invoices themselves. The suppliers working capital management is of particular importance. Is the supplier’s net working capital positive or negative? How have its metrics trended in the past? What is the supplier’s collection history with this specific buyer? Also, what is its history of commercial disputes and/or failures to deliver?
+
+### 3. Industry and Country
+In addition to the supplier’s and buyer’s individual credit risk, the market impact on pricing by industry and country is another important factor. While some industries have a robust, constant supply chain, other industries may be subject to seasonality or prone to disruption. Each country carries with it some amount of systematic risk that trickles down to suppliers and its ability to pay. Further, the recovery of defaulted amounts or the ability to collect an overdue payment often depends on a country’s legal system.
+
+### 4. History 
+There is inherent bias baked into repeating relationships that can be either positive or negative. If a Lender has some track record with a supplier/borrower, then the impact of a continuing relationship will work its way into the pricing. For instance, if an efficiently run supplier has a long track record, with invoices based on contracts with stable counterparties, they will get favorable pricing on its invoice population.
+
+### 5. Portfolio Volume
+The most important factor when estimating the advance rate/fee for invoices is volume. To a Lender, a large number of invoices means the risk of non-payment is spread across a number of counterparties — more invoices are better than less. If one is pricing a pool of invoices, the statistics around the invoice pool can be considered. One can think of a distribution function where a great sample amount means a smoother distribution. Lower invoice counts mean that collecting on the discrete, single invoice has a more significant impact on the pool’s return.
+
+## Estimating Advance Fee and Rate 
+One of our users is ConsolFreight, which is a SaaS freight technology provider that advances working capital finance to freight forwarders’ operations (e.g. air, ocean, rail, road, multi-modal, etc.) and collects from shippers (e.g. supermarkets, hospitals, office supplies, etc.). The founding team combines 50+ years of experience running successful freight forwarding businesses in the U.S., Latin America, and Europe.
+
+We will use ConsolFreight to take a closer look at how considering all these factors may translate into an advance rate and a fee/interest rate for financing an invoice.
```

Comment:
I'm not sure if it's appropriate to add consolfreight here, maybe we should rename that to just "Asset Originator"?